usecases/auth/authorization/rbac/manager.go

```diff
@@ -169,11 +164,70 @@ func (m *manager) Authorize(principal *models.Principal, verb string, resources
 			return err
 		}
 
-		// TODO audit-log ?
+		perm, err := conv.PathToPermission(verb, resource)
+		if err != nil {
+			return err
+		}
+
+		m.logger.WithFields(logrus.Fields{
+			"action":         "authorize",
```

Comment:
Question/suggestion: I'm not familiar with all the other rbac changes. Do all the rbac related log lines share the same action ? This way we can make it "easy" (not really but still) to filter logs and get all rbac related log lines (change, authorization, etc...).

If that's already the case feel free to ignore, but if not maybe for all rbac log adding something to easily filter them would be nice.

Reply:
yes this function is the aggregate for any AuthZ happens